feature_layers/service-feature-table-manual-cache/README.md

```diff
@@ -1,26 +1,37 @@
-# Service Feature Table (Manual Cache)
+# Service feature table (manual cache)
 
-Request features on demand.
+Display a feature layer from a service using the **manual cache** feature request mode.
 
-![](ServiceFeatureTableManualCache.png)
+![Image of service feature table manual cache](ServiceFeatureTableManualCache.png)
 
-## How to use the sample
+## Use case
 
-Click on the Request Cache button to manually request Features. Returned label displays how many features were returned by the service.
+`ServiceFeatureTable` supports three request modes, which define how features are requested from the service and stored in the local table. The feature request modes have different performance characteristics. Use **manual cache** in scenarios where you want to explicitly control requests for features. See [Table performance concepts](https://developers.arcgis.com/net/latest/wpf/guide/layers.htm#ESRI_SECTION1_40F10593308A4718971C9A8F5FB9EC7D) to learn more.
 
-Note: Maximum of Features returned is set to 1000.
+## How to use the sample
 
-## How it works
+Run the sample and pan and zoom around the map. No features will be requested and displayed automatically. Press the Populate button, and features will display.
 
-How to set `FeatureRequestMode.MANUAL_CACHE` mode:
+## How it works
 
-1. Create a `ServiceFeatureTable` from a URL.
-2. Set request mode of table, `ServiceFeatureTable.setFeatureRequestMode(FeatureRequestMode.MANUAL_CACHE)`.
+1. Set the `ServiceFeatureTable.FeatureRequestMode` property of the service feature table to `MANUAL_CACHE` before the table is loaded.
+2. Load the table.
+3. Call `populateFromServiceAsync()` on the table to request features.
 
 ## Relevant API
 
-* ArcGISMap
 * FeatureLayer
-* MapView
+* FeatureRequestMode
 * ServiceFeatureTable
-* ServiceFeatureTable.FeatureRequestMode
+
+## About the data
+
+The U.S. National Bridge Inventory describes 600,000 bridges in the United States. The sample uses [US Bridges](https://arcgisruntime.maps.arcgis.com/home/item.html?id=250b103a722c4e1ea71e562eac61be1b), a modified copy of the U.S. National Bridge Inventory hosted on ArcGIS Online. The sample opens with an initial visible extent centered over Bridgeport, CT.
```

Comment:
Very good catch, thank you! Will have to raise this in the common design, as other API also seem to use the trees in San Francisco SFT.
For our sample I'll keep the old data for now, since there's a query in the code and I don't want to spend too much time adapting the query to a new dataset.